src/ExtendedView.js

```diff
@@ -5,14 +5,26 @@ function ExtendedView(props) {
   console.log(singleMovie)
 
   return(
-    <div className="extended-view">
-      <img
-        className="expanded-movie"
-        key={singleMovie.id}
-        src={singleMovie.backdrop_path}
-      />
-      <button onClick={() => props.changeExtendedState()}>BACK</button>
-    </div>
+    <div className="extended-page">
+      <button className="back" onClick={() => props.changeExtendedState()}>BACK</button>
+        <div className="movie-poster">
+          <div
+            className="movie-image"
+            style={{backgroundImage: `url(${singleMovie.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}>
+            <p className="title">{singleMovie.title}</p>
+            <button className="rating">Rating: {((singleMovie.average_rating/10)*100).toFixed(0)}%</button>
+          </div>
+        </div>
+          <section className="movie-info">
+            <div className="date-time">
+              <p className="release-date">{singleMovie.release_date.split("").slice(0, 4).join("")}</p>
```

Comment:
This is nice logic to get the proper date format we wanted.

We could pop this out into a helper function outside of the return if we wanted as well, but this works great!